.github/workflows/cleanup-releases.yml

```diff
@@ -0,0 +1,39 @@
+name: Delete Old Release Assets
+on:
+  schedule:
+    - cron: '0 18 * * 1' # Weekly on Monday 18:00 UTC
+  workflow_dispatch:
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v8
+        with:
+          script: |
```

Comment:
I haven't fully tested these GitHub actions so there might be a follow up PR once they execute.